Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM golang:buster as builder
+LABEL io.hockeypuck.temp=true
```

Comment:
Why temp? Or the backward java-style name? I haven't used Docker image labels much, I'm +1 with whatever is most conventional.

Reply:
Java-style hierarchical naming is conventional. From https://docs.docker.com/config/labels-custom-metadata/ : 

> Authors of third-party tools should prefix each label key with the reverse DNS notation of a domain they own, such as com.example.some-label.

The `temp` label indicates that the build docker container is not required after it finishes, so it can be cleaned up to save resources (see contrib/docker-compose/standalone/README.md). I'm relying on the label rather than the container name so that it is generic.